ext/mysql2/mysql2_ext.h

```diff
@@ -11,10 +11,8 @@ void Init_mysql2(void);
 
 #ifdef HAVE_MYSQL_H
 #include <mysql.h>
-#include <mysql_com.h>
```

Comment:
`mysql_com.h` has been included since the very beginning, and has probably always been superfluous it looks - but I am loathe to remove it at a micro release.